src/main/scala/com/gu/googleauth/auth.scala

```diff
@@ -150,7 +150,7 @@ object AntiForgeryChecker {
   @deprecated("You can use this method if you never rotate your Play Application secret, but that's not a good security practice.\n" +
     "Use https://github.com/guardian/play-secret-rotation and the vanilla `AntiForgeryChecker` constructor","0.7.7")
   def borrowSettingsFromPlay(httpConfiguration: HttpConfiguration): AntiForgeryChecker =
-    AntiForgeryChecker(InitialSecret(httpConfiguration.secret), signatureAlgorithmFromPlay(httpConfiguration))
```

Comment:
In `play-secret-rotation` v0.15 the secret text is no longer wrapped in Play's `SecretConfiguration` case class (which gained additional fields in Play v2.7) - we just use `String` instead.